sprint/.ddev/config.yaml

```diff
@@ -1,17 +1,67 @@
-APIVersion: "1"
+APIVersion: v1.0.0
 name: sprint-[ts]
 type: drupal8
 docroot: drupal8
 php_version: "7.2"
 router_http_port: "8080"
 router_https_port: "8443"
+xdebug_enabled: true
```

Comment:
I don't think this should be there by default, slows down performance quite a lot.

Reply:
I'll revert it. I changed it because it would be nice as default during a sprint, but then again, not everyone is going to contribute or debug code at a sprint.